test/api_nutripatrol_create_flag.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:math';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/src/utils/nutripatrol_types.dart';
+import 'package:test/test.dart';
+
+void main() {
+  setUpAll(() {
+    OpenFoodAPIConfiguration.userAgent = UserAgent(
+      name: 'openfoodfacts-dart',
+      version: '1.0.0',
+      url: '',
+    );
+  });
+  group('$OpenFoodAPIClient create Nutripatrol flag', () {
+    test('Create a valid Nutripatrol flag', () async {
+      final random = Random();
+      final String randomNumber = random.nextInt(1000000).toString();
+      final String barcode = 'sdk_dart_test_$randomNumber';
+      final String userId = 'sdk_dart_test_$randomNumber';
+
+      final MaybeError<CreateNutripatrolFlag> response =
```

Comment:
You're about to store junk in your production server.
I would definitely recommend using a test server.

Reply:
We don't have a preprod for nutripatrol

Reply:
> We don't have a preprod for nutripatrol

We'll see if the prod gets very messy, then.